graphql/resolvers.go

```diff
@@ -1559,6 +1559,7 @@ func (r *queryResolver) CommitQueue(ctx context.Context, id string) (*restModel.
 		} else if utility.FromStringPtr(item.PatchId) != "" {
 			patchId = utility.FromStringPtr(item.PatchId)
 		}
+
```

Comment:
is this line intentional?

Reply:
Not really but I think I was reading the code and put it there to make reading it easier 🤷